packages/atomic/src/components/atomic-breadbox/atomic-breadbox.tsx

```diff
@@ -253,8 +323,22 @@ export class AtomicBreadbox implements InitializableComponent {
         <span part="label" class="font-bold p-2 pl-0 with-colon">
           {this.bindings.i18n.t('filters')}
         </span>
-        <ul class="flex flex-wrap gap-1">{this.renderBreadcrumbs()}</ul>
+        <div class="relative flex-grow">
+          <ul
+            class={`flex gap-1 ${
+              this.isCollapsed
+                ? 'flex-nowrap overflow-hidden absolute w-full'
+                : 'flex-wrap'
+            }`}
+          >
+            {this.renderBreadcrumbs()}
+          </ul>
+        </div>
       </div>
     );
   }
+
+  public componentDidRender() {
+    this.adaptBreadcrumbs();
+  }
```

Comment:
If the new changes were using JSX, then everything would be managed by the `render` call, removing the need for this life cycle method?

If there is a benefit of the current approach though, I would prefer if this life cycle method was closer to the top of the class (e.g. next to `disconnectedCallback`).